### DISCUSSION TITLE
Feature request: individual notifications for activity within pull request

### DISCUSSION THREAD

**Discussion by OliverJAsh** [community/community#23294] [New to GitHub]
<p>When I receive multiple comments on a pull request, I would like to have a notification for each one so I don’t forget to view/reply to all. Currently I only receive one notification for the most recent activity.</p>
<p>When there is only one notification to signify “activity” on a pull request, it is very easy to miss a comment, especially if it was a reply to a thread/discussion where that code has since been collapsed (e.g. because it’s resolved, outdated, or behind a lazy loaded section).</p>
<p>Email notifications provide this, so I’m currently using those, but I would like to be able to use the web UI. This is the only thing that’s stopping me from switching.</p>

**Comment by ghost** [comment, marked as answer]
<p>Hey <a class="mention" href="/oliverjash">@oliverjash</a>! Thanks for reaching out with your feedback. </p>
<p>Our product team is keen to hear this sort of feedback and we encourage you to share it with them here:</p>
<p><a href="https://support.github.com/contact/feedback" rel="nofollow noopener">https://support.github.com/contact/feedback</a></p>
<p>While we can’t guarantee a response to every feature request, we do appreciate you passing this on for us to consider. </p>
